src/state/slices/opportunitiesSlice/resolvers/thorchainsavers/utils.ts

```diff
@@ -150,7 +150,7 @@ export const getThorchainSaversPosition = async ({
   )
 
   if (!accountPosition) {
-    throw new Error(`No THORCHain savers position in ${assetId} pool for accountId ${accountId}`)
+    return null
```

Comment:
The guts of this - don't throw, this isn't an exception but an actual feature (no position for a given opportunity is def. valid)